test/Helsenorge.Messaging.Tests/Helsenorge.Messaging.Tests.csproj

```diff
@@ -29,4 +29,16 @@
         <ProjectReference Include="..\..\src\Helsenorge.Messaging\Helsenorge.Messaging.csproj" />
         <ProjectReference Include="..\Helsenorge.Registries.Tests\Helsenorge.Registries.Tests.csproj" />
     </ItemGroup>
+
+    <ItemGroup>
+      <None Update="Files\MetadataTests_Apprec10.xml">
```

Comment:
Use a wildcard approach to these, instead of 1 per item.
`<None Update="Files\*.xml">
        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
      </None>`